content/package.json

```diff
@@ -30,6 +30,6 @@
         "sync-pom-version-to-package": "^1.6.0"
     },
     "dependencies": {
-        "@adobe/adobe-client-data-layer": "1.1.3"
+        "@adobe/adobe-client-data-layer": "1.1.5"
```

Comment:
the latest release on the Github repo is 1.1.3: should it be updated to 1.1.5?
https://github.com/adobe/adobe-client-data-layer/releases/tag/1.1.3

Reply:
The newer releases just don't have proper release notes: https://github.com/adobe/adobe-client-data-layer/releases/tag/v1.1.5